suitcase/spec.py

```diff
@@ -325,10 +372,7 @@ def __len__(self):
     def __eq__(self, obj):
         if not isinstance(obj, type(self)):
             return False
-        return (self.specfile == obj.specfile and
-                self.raw_scan_data == obj.raw_scan_data and
-                self.md == obj.md and
-                self.scan_data.equals(obj.scan_data))
+        return _md5(self.raw_scan_data) == _md5(obj.raw_scan_data)
```

Comment:
Are these called often? Is `raw_scan_data` ever changed? 
Depending on the answers to the above, you may want to only calculate the md5sum once as these calls seem pretty expensive.